recipes/trustregion/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "trustregion" %}
+{% set version = "1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trustregion-{{ version }}.tar.gz
+  sha256: 243574345835f8c59c645c9384da015cba67912b1298cb2aa8ada25f3c3918cd
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - trustregion
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/lindonroberts/trust-region
+  summary: Trust-region subproblem solvers for nonlinear/nonconvex optimization
+  license: GPL-3.0-only
+  license_file: LICENSE
```

Comment:
I opened a PR upstream to include the license file in the source distribution (https://github.com/lindonroberts/trust-region/pull/5). Please add a comment in the `meta.yaml` above the `license_file` line:

```
# License file manually packaged. See: https://github.com/lindonroberts/trust-region/pull/5
license_file: LICENSE
...
```

The comment is a reminder to check the status of the license file in the source distribution. If on a subsequent release the license file is in the source distribution, you can remove the manually packaged copy and the comment.

Reply:
Thanks, updated. (and followed the same advice for #17021)